account_authentications.go

```diff
@@ -0,0 +1,141 @@
+package aiven
+
+import (
+	"errors"
+	"time"
+)
+
+type (
+	// AccountAuthenticationsHandler Aiven go-client handler for Account Authentications
+	AccountAuthenticationsHandler struct {
+		client *Client
+	}
+
+	// AccountAuthenticationMethod represents account authentication method
+	AccountAuthenticationMethod struct {
```

Comment:
I'm not sure whether this matters; not all of these fields can be posted in (for example, `saml_certificate_issuer` is in responses but not allowed when updating the authentication method).

Reply:
That is why **saml_certificate_issuer** has **omitempty** option `json:"saml_certificate_issuer,omitempty"`. It means if it is empty, it won't be included to a Request JSON object. During creation, this field should be empty and will be omitted if not API will retrieve an error.

Reply:
It will allow us to use the same object for request and responses instead of having two separate once with almost the same preset of fields.